roles/db-copy/tasks/active-copy.yml

```diff
@@ -270,6 +287,14 @@
       become: true
       become_user: "{{ oracle_user }}"
 
+  always:
```

Comment:
It's also possible that we had a failure before the file was created;  consider either a block for just the template creation and active copy, or an ignore_errors on the deletion.